src/t8_geometry/t8_geometry_implementations/t8_geometry_zero.hxx

```diff
@@ -86,6 +86,23 @@ struct t8_geometry_zero: public t8_geometry
   t8_geom_evaluate_jacobian (t8_cmesh_t cmesh, t8_gloidx_t gtreeid, const double *ref_coords, const size_t num_coords,
                              double *jacobian) const;
 
+  /**
+   * \param[in] forest            The forest of the element.
+   * \param[in] ltreeid           The local tree id of the element's tree
+   * \param[in] element           The element
+   * \param[in] points            points to check
+   * \param[in] num_points        Number of points to check
+   * \param[in, out] is_inside    Array to fill with flags whether the point is inside or not
+   * \param[in] tolerance         Tolerance of the inside-check
+   */
+  virtual void
+  t8_geom_point_batch_inside_element (t8_forest_t forest, t8_locidx_t ltreeid, const t8_element_t *element,
+                                      const double *points, const int num_points, int *is_inside,
+                                      const double tolerance)
+  {
+    SC_ABORTF ("Function not yet implemented");
```

Comment:
```suggestion
const int zeros[3] = { 0 }; 
for (int i_point = 0; i_point < num_points; ++i_point)
{
  const int offset = i_point * T8_ECLASS_MAX_DIM;
  is_inside[i_point] = t8_vertex_point_inside(zeros, points + offset, tolerance)
}
```

Reply:
The zero-geometry does not inherit from `t8_geometry_with_vertices' , therefore I don't think we should implement it this way.

Reply:
Yes it does not inherit from it, but in the end it is still used to compute vertices of the mesh. How would you do it?